### DISCUSSION TITLE
后期是否会收费？

### DISCUSSION THREAD

**Discussion by wgetnz** [he3-app/he3-feedback#17] [General]
RT.

**Comment by zhenweiwang1990** [comment]
是的，这是毋庸置疑的。我们是一家创业阶段的商业公司，我们希望服务好开发者的同时获取合理的收入来支持公司的发展。

但，就目前阶段来说，我们还没有规划好非常细致的收费计划，目前和未来相当长的一段时间内公司和产品的目标都会是以服务个体开发者为主，并确保即便未来有收费版本推出，用户仍然有合理的可免费使用的功能和配额。

以上是很傻的一个回复，看起来套话很多。

作为 CEO，我跟你说句心里话：“我也不知道这东西未来能否收钱，我们现在尽力服务开发者，能收到并维持产品高质量发展最好，收不到公司破产了我就把产品开源出去。”

**Reply by wgetnz** [reply]
很棒的工具，后期价格合理的话，我应该会为它付费

**Reply by liudaobenyuan** [reply]
作为 CEO，我跟你说句心里话：“我也不知道这东西未来能否收钱，我们现在尽力服务开发者，能收到并维持产品高质量发展最好，收不到公司破产了我就把产品开源出去。”  这个回复很中肯！
